packages/framework/client-logger/external-telemetry/src/test/containerTelemetry.spec.ts

```diff
@@ -169,121 +149,23 @@ describe("External container telemetry", () => {
 		expect(actualTelemetryEvent.properties.containerId).to.be.a("string").with.length.above(0);
 	});
 
-	it("Emitting 'closed' system event produces expected ContainerClosedTelemetry", () => {
-		startTelemetry(telemetryConfig);
-
-		mockContainer.close();
-
-		// Obtain the events from the method that the spy was called with
-		const actualTelemetryEvent = trackEventSpy.getCall(0).args[0];
-		const expectedEvent = {
-			name: ContainerTelemetryEventNames.CLOSED,
-			properties: {
-				eventName: ContainerTelemetryEventNames.CLOSED,
-				documentId: mockContainer.resolvedUrl?.id,
-				clientId: mockContainer.clientId,
-				containerId: actualTelemetryEvent.properties.containerId,
-			} as ContainerClosedTelemetry,
-		};
-
-		expect(expectedEvent).to.deep.equal(actualTelemetryEvent);
-		// We won't know what the container UUID will be but we can still check that it is defined.
-		expect(actualTelemetryEvent.properties.containerId).to.be.a("string").with.length.above(0);
-	});
-
-	it("Emitting 'closed' system event with an error produces expected ContainerClosedTelemetry", () => {
-		startTelemetry(telemetryConfig);
-
-		const containerError: ICriticalContainerError = {
-			errorType: "unknown error",
-			message: "An unknown error occured",
-			stack: "example stack error at line 52 of Container.ts",
-		};
-
-		mockContainer.close(containerError);
-
-		// Obtain the events from the method that the spy was called with
-		const actualTelemetryEvent = trackEventSpy.getCall(0).args[0];
-		const expectedEvent = {
-			name: ContainerTelemetryEventNames.CLOSED,
-			properties: {
-				eventName: ContainerTelemetryEventNames.CLOSED,
-				documentId: mockContainer.resolvedUrl?.id,
-				clientId: mockContainer.clientId,
-				containerId: actualTelemetryEvent.properties.containerId,
-				error: containerError,
-			} as ContainerClosedTelemetry,
-		};
-		expect(expectedEvent).to.deep.equal(actualTelemetryEvent);
-		// We won't know what the container UUID will be but we can still check that it is defined.
-		expect(actualTelemetryEvent.properties.containerId).to.be.a("string").with.length.above(0);
-	});
-
-	it("Emitting 'attaching' system event produces expected ContainerAttachingTelemetry", () => {
+	it("Emitting 'disposed' system event produces expected ContainerClosedTelemetry", () => {
```

Comment:
Let's add one for a disposed event with an error too.

Reply:
Updated in commit 20a366b